.poshchan/settings.json

```diff
@@ -1,7 +1,15 @@
 {
-    "build_targets": {},
+    "build_targets": {
+        "ci": "docker-PR",
+        "all": [
+            "docker-PR"
+        ]
+    },
     "authorized_users": [
         "SteveL-MSFT",
-        "TravisEz13"
+        "TravisEz13",
+        "anmenaga",
+        "RDIL",
```

Comment:
Please don't include anyone outside Microsoft.  At least until we have a policy for that.